src/modules/dashboard/components/ActionsPage/staticMaps.ts

```diff
@@ -3,6 +3,13 @@ import { ColonyAndExtensionsEvents, ColonyActions } from '~types/index';
 
 import { STATUS } from './types';
 
+export enum ActionPageDetails {
+  fromDomain = 'fromDomain',
+  toDomain = 'toDomain',
+  toRecipient = 'toRecipient',
+  amount = 'amount',
+}
+
```

Comment:
Please rename this to start with an uppercase letter in order to keep naming consistent.

Eg:
```
FromDomain = 'FromDomain',
```